core/domain/stats_services.py

```diff
@@ -68,7 +82,21 @@ def get_visualizations_info(exploration_id, state_name):
 
 # TODO(bhenning): Test
 def get_versions_for_exploration_stats(exploration_id):
-    """Returns list of versions for this exploration."""
+    """Returns a list of string, each string representing version of the given
```

Comment:
... each string representing a version of the given exploration_id for which statistics data exists. These ...